### PR TITLE
New version: ArobasMusic.GuitarPro.8 version 8.1.1-17

### LINKED ISSUE #133748: [Package Issue]: ArobasMusic.GuitarPro.8 having issues with 8.1.1.17 vs. 8.1.1-17
### Please confirm these before moving forward

- [X] I have searched for my issue and not found a work-in-progress/duplicate/resolved issue.
- [X] I have not been informed if the issue is resolved in a preview version of the winget client.

### Category of the issue

Installation issue.

### Brief description of your issue

I have installed the latest version of Guitar Pro 8 (Internally saying in the about-dialog: Version 8.1.1 - Build 17) and winget is reporting the "current version" as **8.1.1-17**, while configuration is saying **8.1.1.17** is current

### Steps to reproduce

winget upgrade
(while latest Guitar Pro 8 ist installed)

### Actual behavior

It shows, that an update is required

```
Name                                                   ID                            Version    Verfügbar Quelle
----------------------------------------------------------------------------------------------------------------
Guitar Pro 8                                           ArobasMusic.GuitarPro.8       8.1.1-17   8.1.1.17  winget
```

### Expected behavior

It does not report an update is required

### Environment

```raw
Windows-Paket-Manager v1.6.3482
Copyright (c) Microsoft Corporation. Alle Rechte vorbehalten.

Windows: Windows.Desktop v10.0.22631.3007
Systemarchitektur: X64
Paket: Microsoft.DesktopAppInstaller v1.21.3482.0

WinGet-Verzeichnisse
-----------------------------------------------------------------------------------------------------------------------
Protokolle                                   %LOCALAPPDATA%\Packages\Microsoft.DesktopAppInstaller_8wekyb3d8bbwe\Local…
Benutzereinstellungen                        %LOCALAPPDATA%\Packages\Microsoft.DesktopAppInstaller_8wekyb3d8bbwe\Local…
Verzeichnis für portierbare Links (Benutzer) %LOCALAPPDATA%\Microsoft\WinGet\Links
Portables Linkverzeichnis (Computer)         C:\Program Files\WinGet\Links
Portierbarer Paketstamm (Benutzer)           %LOCALAPPDATA%\Microsoft\WinGet\Packages
Portierbarer Paketstamm                      C:\Program Files\WinGet\Packages
Portierbares Paketstamm (x86)                C:\Program Files (x86)\WinGet\Packages
Installationsprogrammdownloads               %USERPROFILE%\Downloads

Links
-----------------------------------------------------------------------------------------
Datenschutzerklärung              https://aka.ms/winget-privacy
Lizenzvereinbarung                https://aka.ms/winget-license
Hinweise von Drittanbietern       https://aka.ms/winget-3rdPartyNotice
Startseite                        https://aka.ms/winget
Windows Store-Nutzungsbedingungen https://www.microsoft.com/en-us/storedocs/terms-of-sale

Administratoreinstellung                  Status
-----------------------------------------------------
LocalManifestFiles                        Deaktiviert
BypassCertificatePinningForMicrosoftStore Deaktiviert
InstallerHashOverride                     Deaktiviert
LocalArchiveMalwareScanOverride           Deaktiviert
```

### Screenshots and Logs

[WinGet-2024-01-11-22-23-50.705.log.zip](https://github.com/microsoft/winget-pkgs/files/13908593/WinGet-2024-01-11-22-23-50.705.log.zip)

### PR DESCRIPTION
**Pull request by Exorcism0666**
### Pull request has been created with [Komac](https://github.com/russellbanks/Komac) v1.11.0 :rocket:

- Resolve: #133748

 ###### Microsoft Reviewers: [Open in CodeFlow](https://microsoft.github.io/open-pr/?codeflow=https://github.com/microsoft/winget-pkgs/pull/136051)

### CODE REVIEW

**Comment by Exorcism0666** [comment]
+1.0GB

**Comment by stephengillie** [comment]
> Sequence contains no elements

 - https://github.com/microsoft/winget-pkgs/issues/133371

(Automated response - build 643.)

**Comment by stephengillie** [comment]
Automatic Validation ended with:
> 2024-01-24T17:51:19.5994622Z Processing manifest: manifests\a\ArobasMusic\GuitarPro\8\8.1.1.17
 2024-01-24T17:51:19.6010753Z -----------------------------------------------------
 2024-01-24T17:51:19.6019207Z Status: Started
 2024-01-24T17:51:19.6027551Z Manifest file is Deleted
 2024-01-24T17:51:19.8667242Z Status: Completed
 2024-01-24T17:51:19.8678422Z ##[error] Result: Failed
 2024-01-24T17:51:19.8733172Z ##[error] Sequence contains no elements
 2024-01-24T17:51:19.9325259Z ##[error]Manifest Validation Failed

(Automated response - build 643.)

**Comment by Exorcism0666** [comment]
## For moderators

This account is automated by Github Actions and the source code was created by CoolPlayLin. If you have any questions about any pull request, don't hesitate to ping @Exorcism0666, I'll get a notification.
>[!important]
>Please carefully review these Pull Request before merging. If it is a Pull Request for removing incorrect content and the URLs are issue, free upon manual checking, please close this Pull Request directly. (It is best to inform Exorcism0666 of the closure.)

良い一日をお過ごしください！

**Comment by stephengillie** [comment]
Automatic Validation ended with:
> 2024-01-24T17:51:19.5994622Z Processing manifest: manifests\a\ArobasMusic\GuitarPro\8\8.1.1.17
 2024-01-24T17:51:19.6010753Z -----------------------------------------------------
 2024-01-24T17:51:19.6019207Z Status: Started
 2024-01-24T17:51:19.6027551Z Manifest file is Deleted
 2024-01-24T17:51:19.8667242Z Status: Completed
 2024-01-24T17:51:19.8678422Z ##[error] Result: Failed
 2024-01-24T17:51:19.8733172Z ##[error] Sequence contains no elements
 2024-01-24T17:51:19.9325259Z ##[error]Manifest Validation Failed

(Automated response - build 679.)

**Comment by stephengillie** [comment]
> Sequence contains no elements

 - https://github.com/microsoft/winget-pkgs/issues/133371

(Automated response - build 679.)

**Comment by stephengillie** [comment]
Automatic Validation ended with:
>  2024-02-20 12:39:35.499   Key: HKEY_LOCAL_MACHINE\SOFTWARE\Microsoft\Windows\Windows Error Reporting\LocalDumps\GuitarPro.exe 2024-02-20 12:39:51.452   Exception message:                           CreateProcess failed; code 216.

(Automated response - build 788.)

File: manifests/a/ArobasMusic/GuitarPro/8/8.1.1-17/ArobasMusic.GuitarPro.8.installer.yaml [hunk 1]
```diff
@@ -1,9 +1,13 @@
...
 PackageIdentifier: ArobasMusic.GuitarPro.8
-PackageVersion: 8.1.1.17
+PackageVersion: 8.1.1-17
 InstallerType: inno
+ReleaseDate: 2023-10-05
+InstallerSwitches:
+  Silent: /sp- /verysilent /norestart
+  SilentWithProgress: /sp- /silent /norestart
 Installers:
 - Architecture: x86
```
Comment (stephengillie) [inline manifests/a/ArobasMusic/GuitarPro/8/8.1.1-17/ArobasMusic.GuitarPro.8.installer.yaml:12]:
```suggestion
- Architecture: x64
```

**Review by stephengillie — CHANGES_REQUESTED** [review]
From the error, I'm guessing this is a 64-bit application. As the previous version was `x86`, I'm guessing the developer might have increased the architecture requirement with this version.

**Review by stephengillie — APPROVED** [review]

**Pull request merged**